### ISSUE TITLE
Bridging tutorial and sample

### ISSUE THREAD

**Issue by vranki** [mautrix/python#37]
Currently documentation is quite minimal and getting started with writing bridges is quite difficult.

My suggestion is to write some kind of tutorial explaining the most important classes, their relations and functions.

Another helpful thing would be a sample bridge that would have a minimal implementation of a working bridge with static users. It could also be used as template for writing new bridges.

**Comment by weeman1337** [comment]
Same here. I wonder how to get started. If you @vranki got something working please leave a comment here.

**Comment by vranki** [comment]
I started sponsoring @tulir and decided this would be my "favorite" issue to fix. Sounds trivial but with good documentation and examples we could have more new bridges and helping hands to work on existing bridges.

**Comment by gabrc52** [comment]
Should we at least keep the issue open and write documentation ourselves? Even if tulir & Beeper et al does not have availability to write documentation

**Comment by tulir** [comment]
No, the bridge module is likely to be removed as soon as the Telegram and Google Chat rewrites are finished. There are already docs for writing bridges with bridgev2 https://mau.fi/blog/megabridge-twilio/
